generated/nidcpower/nidcpower_service.cpp

```diff
@@ -2097,6 +2173,21 @@ namespace nidcpower_grpc {
     }
   }
 
+  //---------------------------------------------------------------------
+  //---------------------------------------------------------------------
+  ::grpc::Status NiDCPowerService::FetchMultipleLcr(::grpc::ServerContext* context, const FetchMultipleLcrRequest* request, FetchMultipleLcrResponse* response)
```

Comment:
Are these new service methods intended to be implemented in the `nidcpower_service.custom.cpp` file at a later point or should that be included in this change?

Reply:
yes, the plan is to add them in subsequent PRs, in nidcpower_service.custom.cpp